naucse/templates/index.html

```diff
@@ -9,28 +9,28 @@
     <div class="card card-inverse space motivation">
       <img class="card-img" src="{{url_for('static', filename='pycon2016cz.png')}}" alt="Card image">
       <div class="card-img-overlay">
-        <p class="responsive-text card-text center">Nauč se Python, staň se superhrdinou nebo superhrdinkou a zachraň svět.<br>
-            Python je moderní programovací jazyk. Je univerzální – pohání weby i rakety.
-            Dobře se čte a dá se velice rychle naučit.</p>
+        <p class="responsive-text card-text center">Nauč se Python, staň se superhrdinou nebo superhrdinkou a&nbsp;zachraň svět.<br>
```

Comment:
And typographicaly it's ok to break line after „a“ and „i“. But I don’t like it either.